specification/communication/data-plane/Microsoft.CommunicationServicesAdministration/ga/2021-03-07/phonenumbers.json

```diff
@@ -0,0 +1,932 @@
+          "description": "The error code.",
+          "type": "string"
+        },
+        "message": {
+          "description": "The error message.",
+          "type": "string"
+        },
+        "target": {
+          "description": "The error target.",
+          "type": "string",
+          "readOnly": true
+        },
+        "details": {
+          "description": "Further details about specific errors that led to this error.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/CommunicationError"
+          },
+          "readOnly": true
+        },
+        "innererror": {
+          "$ref": "#/definitions/CommunicationError"
+        }
+      }
+    },
+    "CommunicationErrorResponse": {
+      "description": "Represents the wrapper around the Communication Services error",
+      "required": [
+        "error"
+      ],
+      "type": "object",
+      "properties": {
+        "error": {
+          "$ref": "#/definitions/CommunicationError"
+        }
+      }
+    },
+    "PhoneNumberUpdateRequest": {
+      "description": "Represents a phone number update request",
+      "type": "object",
+      "properties": {
+        "callbackUrl": {
+          "description": "The webhook URL on the phone number configuration.",
+          "type": "string"
+        },
+        "applicationId": {
+          "description": "The application id of the application to which to configure",
+          "type": "string"
+        }
+      }
+    },
+    "AcquiredPhoneNumbers": {
+      "description": "A wrapper around a list of acquired phone numbers",
+      "type": "object",
+      "properties": {
+        "phoneNumbers": {
+          "description": "Represents a list of phone numbers",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AcquiredPhoneNumber"
+          }
+        },
+        "nextLink": {
+          "description": "Represents the URL link to the next page of phone number results",
+          "type": "string"
+        }
+      }
+    },
+    "PhoneNumberLongRunningOperation": {
+      "description": "Represents the status of a long running operation",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Represents the status of a long running operation",
+          "enum": [
+            "NotStarted",
+            "Running",
+            "Succeeded",
+            "Failed"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "PhoneNumberLongRunningOperationStatusCode",
+            "modelAsString": true
+          }
+        },
+        "resourceLocation": {
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "error": {
+          "$ref": "#/definitions/CommunicationError"
+        },
+        "id": {
+          "type": "string"
+        },
+        "longRunningOperationType": {
```

Comment:
Either call this `kind` or `operationType`. If we call it "long running" the url should be /phoneNumbers/longRunningOperations/ too and that's unnecessarily long 🙂

Reply:
Done.